### DISCUSSION TITLE
Indiana Traffic Signal Hi Resolution Data Codes

### DISCUSSION THREAD

**Discussion by gmonk** [udotdevelopment/ATSPM#110] [Q&A] [answered]
Is there a list of codes that are used for the ATSPM metrics ?  I see there are over 100 codes and i'm wondering if there are only a few codes that are actually used when determining metrics and if so could you provide these.

Thanks,

**Comment by gault00** [comment]
The original codes (enumerations) are here: https://docs.lib.purdue.edu/cgi/viewcontent.cgi?article=1002&context=jtrpdata

An update was done in 2019/2020 but I don’t think any of the new codes are implemented in the ATSPM open-source software yet. https://docs.lib.purdue.edu/jtrpdata/4/

**Comment by gmonk** [comment]
I was more referencing which codes within theses lists are actively being used for the metrics provided by ATSPM ?  You mentioned none of the new codes are implemented so which codes are being implemented to provide  reports such as Perdure Split Failure, Perdue Phase Termination...etc

**Comment by taylormd3** [comment, marked as answer]
Please see the "Methods and Assumptions" report that will show all the active enumerations being used for the various metrics.

https://drive.google.com/file/d/1AULeek1Qp-BS_HACqXsfzty94rJlXj6A/view?usp=sharing

**Comment by gmonk** [comment]
Exactly what I was looking for thank you...
